### ISSUE TITLE
auto preview generation

### ISSUE THREAD

**Issue by trinityss** [LJ3D/AssetLibraryTools#9]
Hi,

Awesome addon you created! Exactly what I need!

I've tried the batch import PRB textures function and it's doing a good job. But the preview is not auto generated. I have to click on every single asset followed by a "generate preview" click. Can this preview be created during import?

Thx!

**Comment by lucianjames** [comment]
Are you talking about the previews in the asset browser? or the mini previews you get in the materials menu?

**Comment by trinityss** [comment]
I'm referring to the preview in the asset browser.

**Comment by lucianjames** [comment]
Ok, looked into it and it can be done pretty simply.
```
for mat in bpy.data.materials:
    if mat.asset_data:
        mat.asset_generate_preview()
```
Ill add a button into assetlibrarytools to do that

**Comment by lucianjames** [comment]
191d3ff7e4c51ec7f9f9ac52f2b31d33ed2b6182

**Comment by trinityss** [comment]
Hi,

Is the fix part of the new 0.2.0 version? I installed that version but there is no change in behavior, the preview isn't generated. Here is a recording: https://drive.google.com/file/d/1cNjAq6xTUrYcJYAmU4EE47cEaddZfkkn/view?usp=sharing

Kr,

**Comment by lucianjames** [comment]
![55390e10c8a683d9de8da489d089262fe1e57811](https://user-images.githubusercontent.com/65134690/138965528-bf9416e4-26cd-4179-89a7-b7ab37deaa4b.png)
You need to press this button (after changing "objects" to "materials")
